libdino/src/service/muc_manager.vala

```diff
@@ -13,6 +13,7 @@ public class MucManager : StreamInteractionModule, Object {
     public signal void left(Account account, Jid jid);
     public signal void subject_set(Account account, Jid jid, string? subject);
     public signal void room_name_set(Account account, Jid jid, string? room_name);
+    public signal void members_only_non_anonymous_occupant_updated(Account account, Jid room, Jid occupant);
```

Comment:
Can we come up with a better name for this? `members_only_non_anonymous` is incredibly long and unreadable, although technically correct. Maybe using a new term like `private room` would be more suitable?

Reply:
"room" might also be an outdated term.. I think groupchat is more common nowadays. As long as it's not "barbecue".